evm/src/cpu/columns/general.rs

```diff
@@ -91,5 +113,32 @@ pub(crate) struct CpuLogicView<T: Copy> {
     pub(crate) output: [T; 16],
 }
 
+#[derive(Copy, Clone)]
+pub(crate) struct CpuJumpsView<T: Copy> {
+    // Assuming a limb size of 32 bits.
+    pub(crate) input0: [T; 8],
+    pub(crate) input1: [T; 8],
+    pub(crate) output: [T; 8],
+
+    pub(crate) input0_upper_sum_inv: T,
+    pub(crate) input0_upper_zero: T,
+
+    pub(crate) dst_valid: T, // TODO: populate this (check for JUMPDEST)
+    pub(crate) dst_valid_or_kernel: T,
+    pub(crate) input0_jumpable: T,
```

Comment:
nit: name seems slightly confusing to me, maybe call it `should_jump` or `jump_to_input1`?